apps/server/src/modules/tldraw/utils/utils.ts

```diff
@@ -0,0 +1,252 @@
+import { setInterval, clearInterval } from 'timers';
+import * as Y from "yjs";
+import * as map from "lib0/dist/map.cjs";
+import * as awarenessProtocol from "y-protocols/dist/awareness.cjs";
+import * as encoding from "lib0/dist/encoding.cjs";
+import * as decoding from "lib0/dist/decoding.cjs";
+import * as syncProtocol from "y-protocols/dist/sync.cjs";
+import { isCallbackSet, callbackHandler } from "./callback";
+import * as debounce from "lodash.debounce";
+import { Configuration } from '@hpi-schul-cloud/commons';
+
+const CALLBACK_DEBOUNCE_WAIT = Configuration.get('FEATURE_TLDRAW_CALLBACK_DEBOUNCE_WAIT') as number ?? 2000;
+const CALLBACK_DEBOUNCE_MAX_WAIT = Configuration.get('FEATURE_TLDRAW_CALLBACK_DEBOUNCE_MAX_WAIT') as number ?? 10000;
+const pingTimeout = Configuration.get('FEATURE_TLDRAW_PING_TIMEOUT') as number ?? 30000;
+// disable gc when using snapshots!
+const gcEnabled = Configuration.get('FEATURE_TLDRAW_GC_ENABLED');
+
+enum wsStatesEnum {
+	wsReadyStateConnecting = 0,
+	wsReadyStateOpen = 1
+}
+
+enum wsMessageTypes {
+	messageSync = 0,
+	messageAwareness = 1
+}
```

Comment:
Please remember to:
- use _UpperCamelCase_ format for enum names and CAPITALIZED_SNAKE_CASE for enum keys,
- not include _Enum_ in enum names
- try to not repeat enum name in enum keys

I would suggest something:
```ts
enum WSConnectionState {
	CONNECTING = 0,
	OPEN = 1,
}

enum WSMessageType {
	SYNC = 0,
	AWARENESS = 1,
}
```
When defining enum I always try to think about how I will read the code that uses them. In my opinion it's easier to read e.g. `WSConnectionState.OPEN` than `wsStatesEnum.wsReadyStateOpen`.